vitrina/helpers.py

```diff
@@ -449,41 +458,137 @@ def get_current_domain(request: WSGIRequest, ensure_secure=False) -> str:
+            else:
+                base_template_content = base_template_content.replace("{" + key + "}", '')
     if not email_template:
         email_subject = email_title = email_title_subject
-        email_content = base_template_content.format(*email_template_keys)
+        email_content = base_template_content
         created_template = EmailTemplate.objects.create(
             created=datetime.datetime.now(),
             version=0,
             identifier=email_identifier,
-            template=base_template_content,
+            template=email_template_to_save,
             subject=_(email_title_subject),
             title=_(email_title)
         )
         created_template.save()
     else:
         email_template = email_template.first()
         email_content = str(email_template.template)
-        email_content = email_content.format(*email_template_keys)
-        email_subject = str(email_template.subject)
+        if email_template_keys:
+            for key in list_keys:
+                if key in email_template_keys.keys():
+                    if email_template_keys[key] is not None:
+                        email_content = email_content.replace("{" + key + "}", email_template_keys[key])
+                else:
+                    email_content = email_content.replace("{" + key + "}", '')
+            email_subject = str(email_template.subject)
+
 
-    return {'email_content': email_content, 'email_subject': email_subject}
+def _get_email_tempate_from_db(name: str) -> EmailTemplate | None:
+    try:
+        return EmailTemplate.objects.get(identifier=name)
+    except EmailTemplate.DoesNotExist:
+        return None
+
+
+def email(
+    recipients: list[str],
+    email_identifier: str,
+    name: str,  # template name
+    context: dict[str, Any] | None = None,
+    *,
+    # Allow user to override email templates via Admin.
+    override: bool = True,
+) -> dict[str, str]:
+    context = context or {}
+    data_dict = {}
+    email_template = None
+    email_send = False
+    subject = None
+    if override:
+        email_template = _get_email_tempate_from_db(email_identifier)
+    if email_template:
+        from unidecode import unidecode
+        subject = email_template.subject
+        template_db = email_template.template
+        template_db = unidecode(template_db)
+        template = Template(template_db)
+        for key in template.nodelist:
+            if type(key).__name__ == 'VariableNode':
+                data_dict.update({key.token.contents: context[key.token.contents]})
+        context = Context(data_dict)
+        content = template.render(context)
+        html_message = markdown.markdown(content)
+    else:
+        template_path = get_template(name)
+        with open(template_path.origin.name, encoding="utf-8") as file:
+            read_data = file.readlines()
+        content = render_to_string(name, context)
+        html_message = markdown.markdown(content)
```

Comment:
HTML message should not include email subject, and email subject is extracted below into a `send_email_title` variable.